content/open source resources and best practices/mentorship/_index.en.adoc

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Open Source Mentorship"
 description: "Documentation and reference material about the UNICEF Open Source Mentorship programme."
+draft: "true"
 
 ---
 
```

Comment:
Let's keep the URL path brief here. `/resources/` will do.

Reply:
Addressed in commits 0eabde03d98eda5e3fb0debea1bfa5781f8440e3 and 547525c0998b1c9de04fe0afeda2cf0bc66abf87.